speedtest.py

```diff
@@ -1823,11 +1860,36 @@ def shell():
                 'be an int' % ', '.join('%s' % s for s in args.server)
             )
 
+        filteredServers = []
+
         if args.server and len(args.server) == 1:
             printer('Retrieving information for the selected server...', quiet)
+        elif args.country:
+            print_('Selecting best server in %s based on latency...' %
+                   args.country.capitalize())
```

Comment:
```suggestion
            printer('Selecting best server in %s based on latency...' %
                   args.country.capitalize(), quiet)
```

here _**and in all other places**_ you should use `printer(..., quiet)` instead of `print_(...)`, otherwise you break `--csv` and `--simple` modes.